addons/l10n_co_pos_details/wizard/pos_details.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class PosDetails(models.TransientModel):
+    _inherit = 'pos.details.wizard'
+
```

Comment:
@nla-odoo The idea was to have a separate wizard as when you call the regular wizard, it will throw an awful error telling that the pos_config_id is required...   It would be better to have a different wizard and to use exclude_products instead of include_products or something where we don't need to bother about passing this info in the other wizard.